packages/styling/src/styles/theme.ts

```diff
@@ -101,7 +101,7 @@ function _loadFonts(theme: ITheme): { [name: string]: string } {
     for (const propName of Object.keys(font)) {
       const name = fontName + propName.charAt(0).toUpperCase() + propName.slice(1);
       let value = font[propName];
-      if (propName === 'fontSize' && value.indexOf('px') < 0) {
+      if (propName === 'fontSize' && value && value.indexOf && value.indexOf('px') < 0) {
```

Comment:
I see `value` is of type `any` in VS Code. Does that mean value could be a number type? If so, would it be better to ensure `value` is a String and then check for 'px'?

My only other comment is I'm used to seeing `=== -1` instead of `< 0` for indexOf. But this is subjective.

Reply:
We should fix the typings etc. in an iteration after rollout is unblocked.

Reply:
I did `<0` solely for the reason that it is less bytes.
fontSize is technically an `any` here because of the way we retrieve it, however it conforms to the type that we've set for it in IStyle (or is it IRawStyle?), which is string or number. (Similar to how padding can be a string or number.)

Reply:
So if the value for fontSize is a number, say `16`, does it need to become `'16px'`, or is it fine as `16`? Because `16.indexOf` would yield a SyntaxError.

Reply:
It's fine as 16. `16.indexOf` doesn't throw:
![image](https://user-images.githubusercontent.com/20691641/63200584-ac1f2500-c036-11e9-8f19-f3f7dcc819c0.png)

Reply:
Yep. My mistake. I meant it will resolve to `false` and won't add 'px'. Not sure if that's needed or not.

Reply:
you are correct! i'll publish another PR